pandas/_libs/tslibs/conversion.pyx

```diff
@@ -90,6 +89,27 @@ cdef class _TSObject:
             return self.value
 
 
+cpdef int64_t pydt_to_i8(object pydt) except? -1:
+    """
+    Convert to int64 representation compatible with numpy datetime64; converts
+    to UTC
+    """
+    cdef:
+        _TSObject ts
+
+    ts = convert_to_tsobject(pydt, None, None, 0, 0)
+
+    return ts.value
+
+
+def i8_to_pydt(int64_t i8, object tzinfo=None):
+    """
+    Inverse of pydt_to_i8
+    """
+    from pandas import Timestamp
```

Comment:
shouldn't this return a ``datetime.datetime``? (exactly, and not a Timestamp)

Reply:
I guess to make the name more accurate.  This is never used, only moved because you asked.

Reply:
if this is not used, then just remove